### DISCUSSION TITLE
ScriptReference/Screen-currentResolution

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#201] [Notes]
# ScriptReference/Screen-currentResolution

https://docs.unity3d.com/ScriptReference/Screen-currentResolution.html

**Comment by mvi** [comment]
Note that `Screen.currentResolution.width` is not guaranteed to be the same as `Screen.width`. In particular `currentResolution` has different behaviour on different platforms - there is more information on this in [this article](http://www.aclockworkberry.com/managing-screen-resolution-and-aspect-ratio-in-unity-3d/).  
In many cases you'll want to use `Screen.width` and `Screen.height` instead of `Screen.currentResolution`.
